contentcuration/contentcuration/frontend/shared/views/AppIcon.vue

```diff
@@ -0,0 +1,35 @@
+<script>
```

Comment:
Just a note: I am not sure when we'll be able to upgrade Vuetify to 2.x though it might be good for us to bear in mind that they've updated `VIcon` to contain `notranslate` class in the new version so we should be able to use it without any customizations needed as soon as we update.

It's possible to preview it by using inspector in examples section https://vuetifyjs.com/en/components/icons#material-design or here's source code https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VIcon/VIcon.ts#L90

Reply:
Oh, awesome! I think we could probably keep this until we upgrade. I had been talking with @jayoshih about this specific issue as something she was looking for a better workaround for the translate issue, in the time being at least.

Reply:
Yes, it makes sense to leave it in for the time being. When I last tried to update to vuetify 2.0, it broke a lot of things for some reason. Once it seems vuetify 2 is stable though, we can definitely revisit

Reply:
icons are also one of the first components I think we'll be able to standardize in the shared component library